frontend/ui/screensaver.lua

```diff
@@ -103,12 +93,10 @@ function Screensaver:_getRandomImage(dir)
     if ok then
         for f in iter, dir_obj do
             -- Always ignore macOS resource forks, too.
-            if lfs.attributes(dir .. f, "mode") == "file" and not util.stringStartsWith(f, "._") then
-                local extension = string.lower(string.match(f, ".+%.([^.]+)") or "")
-                if self.screensaver_provider[extension] then
-                    i = i + 1
-                    pics[i] = f
-                end
+            if lfs.attributes(dir .. f, "mode") == "file" and not util.stringStartsWith(f, "._")
+               and DocumentRegistry:isImageFile(f) then
+                i = i + 1
```

Comment:
Indentation issue on the `and`.

Reply:
To distinguish from the next line.

Reply:
1 space of distinguishment is not really enough :) and non-multiple-of-4 spaces of indentation is bad. I'd go with 8 or 12 (anything more indented, that's what I/we use mostly everywhere), and we will distinguish it from the next line.